tests/openwrt/test_interfaces.py

```diff
@@ -1027,6 +1027,33 @@ def test_parse_dns_list(self):
         o = OpenWrt(native=native)
         self.assertEqual(o.config, self._dns_netjson)
 
+    def test_render_dns_without_interface(self):
+        o = OpenWrt({
+            "dns_servers": ["10.11.12.13", "8.8.8.8"],
+            "dns_search": ["netjson.org", "openwisp.org"],
+        })
+        expected = self._tabs("""package network
+
+config dns 'dns'
```

Comment:
this doesn't look like something that would work on OpenWRT, can you provide the piece of documentation of OpenWRT which explains that this would work? Have you tested this on an actual OpenWRT device?

Reply:
Hmmmm, @nemesisdesign . I just followed standards and conversion used in other tests. I don't have an actual OpenWRT device.

Reply:
@NoumbissiValere You can make an OpenWRT VM on your laptop in the meanwhile for testing. 
This should help: http://openwisp.io/docs/user/configure-device.html#install-openwrt-on-virtualbox

Reply:
Thanks for the help @atb00ker